lib/WUI/wui_api.cpp

```diff
@@ -202,6 +208,7 @@ void load_net_params(ETH_config_t *ethconfig, ap_entry_t *ap, uint32_t netdev_id
         ethconfig->dns2_ip4.addr = config_store().wifi_ip4_dns2.get();
         ethconfig->lan.msk_ip4.addr = config_store().wifi_ip4_mask.get();
         ethconfig->lan.gw_ip4.addr = config_store().wifi_ip4_gateway.get();
+        strlcpy(ethconfig->ntp, config_store().lan_ntp_addr.get_c_str(), DNS_MAX_NAME_LENGTH_EEPROM + 1);
```

Comment:
This looks weird, we are in the wifi section…

Reply:
This is used in the prusa_settings.ini, so should be fine I think

Reply:
I removed the lan_ prefix, as ntp is universal and not limited to lan or wifi